README.md

```diff
@@ -46,7 +42,7 @@ Will have future changes in the next days!
 # 📖 Web page for Gamescom event! <a name="about-project"></a>
 
 > Testing all the information learned in the course in the areas of HTML, CSS and JS to achieve the results shown..
```

Comment:
- [ ] As the last reviewer mentioned, Kindly add the correct project name and description because it seems you used the README file of your HTML & CSS capstone project.